.github/workflows/main.yaml

```diff
@@ -34,6 +34,16 @@ on:
         required: false
         type: string
         default: ''
+      check_artifact_size:
+        description: 'Which workflow to check artifact binary size'
+        required: false
+        type: string
+        default: ''
+      artifact_size_increase_threshold:
+        description: 'Threshold for artifact binary size increase in bytes'
+        required: false
+        type: number
+        default: 5242880 # 5MB
```

Comment:
5MB seems like a very high threshold for a single change. Is that really what we want?

Reply:
Hi @hlwarriner , can you advise on how big the threshold should be for Evergreen libcobalt.so binary increase?

Reply:
Thanks for the advise, used a percentage threshold 2% instead.